src/v2/Components/Artwork/__tests__/Details.jest.tsx

```diff
@@ -207,6 +210,109 @@ describe("Details", () => {
       const html = wrapper.html()
       expect(html).not.toContain("bid")
     })
+
+    describe("lot close info", () => {
+      it("shows the lot is closed if the lot end time has passed and if the sale has cascading end times enabled", async () => {
+        const data = {
+          ...artworkInAuction,
+          sale_artwork: {
+            // @ts-expect-error PLEASE_FIX_ME_STRICT_NULL_CHECK_MIGRATION
```

Comment:
If trying to ignore something in typescript, use `// @ts-ignore`. These flags were added by a codemod when we were adding strict type checking. See [this](https://github.com/artsy/force/blob/main/docs/best_practices.md#avoid-copying-and-try-to-fix--ts-expect-error-please_fix_me_strict_null_check_migration-flags) for more info!